crates/libs/implement/src/lib.rs

```diff
@@ -336,6 +349,18 @@ impl syn::parse::Parse for UseTree2 {
             if input.peek(syn::Token![::]) {
                 input.parse::<syn::Token![::]>()?;
                 Ok(UseTree2::Path(UsePath2 { ident, tree: Box::new(input.parse()?) }))
+            } else if input.peek(syn::Token![=]) {
+                if ident != "TrustLevel" {
+                    return Err(syn::parse::Error::new(ident.span(), "Unrecognized key-value pair"));
+                }
+                input.parse::<syn::Token![=]>()?;
+                let span = input.span();
+                let value = input.call(syn::Ident::parse_any)?;
+                match value.to_string().as_str() {
+                    "Partial" => Ok(UseTree2::TrustLevel(1)),
+                    "Full" => Ok(UseTree2::TrustLevel(2)),
+                    _ => Err(syn::parse::Error::new(span, "`TrustLevel` must be `Partial` or `Full`")),
+                }
```

Comment:
Should we support the explicit specification of `BaseLevel` for folks that want to make it explicitly clear in their code?

```suggestion
                match value.to_string().as_str() {
                    "BaseLevel" => Ok(UseTree2::TrustLevel(0)),
                    "Partial" => Ok(UseTree2::TrustLevel(1)),
                    "Full" => Ok(UseTree2::TrustLevel(2)),
                    _ => Err(syn::parse::Error::new(span, "`TrustLevel` must be `BaseLevel`, `Partial`, or `Full`")),
                }
```

Reply:
That just seems redundant and may confuse developers into thinking that maybe that's required. 🤷